### ISSUE TITLE
One space after and before curly braces when they are on the same line

### ISSUE THREAD

**Issue by lackovic** [editorconfig/editorconfig#359]
Is there a setting for editorconfig file where I can set one space after and before curly braces when they are on the same line?

Example before:
```
{Some test}
{    Some other test}
```

Example after:
```
{ Some test }
{ Some other test }
```

**Comment by xuhdev** [comment]
No, this is too langauge-specific and too detailed that it's hard to get some editors to support this feature.

**Comment by szeredaiakos** [comment]
in .vscode/settings.json
"[lang].format.insertSpaceAfterOpeningAndBeforeClosingNonemptyBraces": true,
where lang is for ex. typescript:
"typescript.format.insertSpaceAfterOpeningAndBeforeClosingNonemptyBraces": true,

in .editorconfig you have (probably about 2 to 3 decades):
spaces_around_brackets
with the value one of: none, inside, outside, both

But those probably wont and shouldn't work for interpolation. Reason being, specific frameworks, languages, ... have their own interpolation of interpolation, I create my own in a shape and form dictated by the project as well.

**Comment by dyadix** [comment]
I think we need a separate place to discuss formatting options since it's out of EditorConfig group's scope to my understanding. It applies not only to this specific option but to a lot of other options as well like brace placement and etc. It would be cool if different IDEs would have some shareable properties which every IDE would understand. Right now some IDEs either already have or are going to have their domain-specific options but sadly they won't be compatible.

**Comment by 5HT2** [comment]
> No, this is too langauge-specific and too detailed that it's hard to get some editors to support this feature.

Intellij supports this option for JS / TS, it would be nice to be able to configure this from an EditorConfig.

> It would be cool if different IDEs would have some shareable properties which every IDE would understand. 

As it stands, VSC, Jetbrains products and most major IDEs by this point just support using an EditorConfig.

Especially when you can specify what file extension this rule can apply to, it would be nice to have this for specifically JS / TS.
